src/custom-elements/src/components/number-input/number-input.tsx

```diff
@@ -72,9 +80,29 @@ export class NumberInput {
    */
   @Prop() state?: string;
 
+  /**
+   * used to show a fixed amount of decimal digits after the decimal point.
+   */
+  @Prop({ reflect: true }) decimals?: number;
```

Comment:
This should not be necessary.

Reply:
Now I see your previous comment, I agree @janibolkvadze  @josecarlosgonzalezv